cpu/atmega_common/posix_unistd.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
```

Comment:
C-files are currently not compiled by doxygen (and are not expected to, since they are no API), so this block is purely cosmetically in doxygen style. Since this does not define a new group `@addtogroup` would be the correct command. Throughout the code-base neither seems to be used in general (and I usually don't include it, because ... see first sentence).